orttraining/orttraining/training_ops/cuda/tensor/gather_elements_grad.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/providers/cuda/cuda_common.h"
+
+namespace onnxruntime {
+namespace cuda {
+
+class GatherElementsGrad final : public CudaKernel {
+ public:
+  GatherElementsGrad(const OpKernelInfo& info) : CudaKernel(info) {
+    ORT_ENFORCE(info.GetAttr<int64_t>("axis", &axis_).IsOK(),
+                "Missing/Invalid 'axis' attribute value");
```

Comment:
Gather Element supports default axis, so does GatherElementGrad

Reply:
yeah. thanks. it's weird the test with default axis can pass. maybe graph schema has the default value.